api/proto/teleport/legacy/types/types.proto

```diff
@@ -1281,6 +1283,24 @@ message ProvisionTokenSpecV2Azure {
   repeated Rule Allow = 1 [(gogoproto.jsontag) = "allow,omitempty"];
 }
 
+// ProvisionTokenSpecV2GCP contains the GCP-specific part of the
+// ProvisionTokenSpecV2.
+message ProvisionTokenSpecV2GCP {
+  // Rule is a set of properties the GCP-ussued token might have to be allowed
+  // to use this ProvisionToken.
+  message Rule {
```

Comment:
Is it possible for us to include small/simple example value for each of these rule matchers in the comment for each field ?